beard/clustering/wrappers.py

```diff
@@ -79,17 +79,32 @@ def __init__(self, method="single", affinity="euclidean",
             The statistics upon which non-singleton i is thresholded.
             See scipy.cluster.hierarchy.fcluster for further details.
 
+         :param scoring_data: string or None
+            The form of the input data
```

Comment:
I would rephrase as "The type of input data to pass into the `scoring` function.`